core/input/input_map.h

```diff
@@ -81,7 +79,7 @@ class InputMap : public Object {
 	static _FORCE_INLINE_ InputMap *get_singleton() { return singleton; }
 
 	bool has_action(const StringName &p_action) const;
-	List<StringName> get_actions() const;
+	TypedArray<StringName> get_actions();
```

Comment:
While this solution isn't perfect, it's probably fair to keep it const if it's going to be const for GDExtension also. Less API discrepancy.

Reply:
I wonder if we can somehow denote this to make it `const` as soon as breaking compat is appropriate

Reply:
While we *could* have some conditional macros in `typedefs.h` to handle this sort of thing, that seems really excessive for this specific case. If this `const` oversight exists, it's likely others exist as well; those could all be handled in a dedicated pass instead of bogging down this specific PR